analyzer/src/main/kotlin/managers/PhpComposer.kt

```diff
@@ -307,7 +307,7 @@ class PhpComposer(
         }.toSet()
 
     private fun parseDeclaredLicenses(packageInfo: JsonNode) =
-        packageInfo["license"]?.mapNotNullTo(sortedSetOf()) { it?.textValue() } ?: sortedSetOf<String>()
+        packageInfo["license"]?.mapNotNullTo(sortedSetOf<String>()) { it?.textValue() } ?: sortedSetOf()
```

Comment:
I think it would be better to define the return type of the function as `SortedSet<String>` instead. Then it wouldn't be `TreeSet<String>`. And I find the "value assigning to the return value" in the commit message confusing.

Reply:
I decided to rewrite the function in a completely different way.